cmd/ci-operator/main.go

```diff
@@ -712,6 +718,27 @@ func parseKeyValParams(input []string, paramType string) (map[string]string, err
 	return params, nil
 }
 
+func appendAdditionalSuffixToTarget(o *options) {
+	if o.targetAdditionalSuffix == "" {
+		return
+	}
+
+	for i, test := range o.configSpec.Tests {
+		for j, target := range o.targets.values {
+			if test.As == target {
```

Comment:
It is unclear to me whether `o.configSpec.Tests` will contain tests that aren't one of the `targets` so this check is just for safety. If someone knows it will not, I can remove the double loop and the check, but it is low cost anyways.

Reply:
@bbguimaraes please confirm, we will merge anyway as we want to test as soon as possible.